src/main/java/uk/gov/ida/matchingserviceadapter/MatchingServiceAdapterModule.java

```diff
@@ -210,8 +216,10 @@ public VerifyMatchingService getVerifyMatchingService(
         @Named("CountryCertificateValidator") Optional<CertificateValidator> countryCertificateValidator,
         X509CertificateFactory x509CertificateFactory,
         MatchingServiceAdapterConfiguration configuration,
-        AssertionDecrypter assertionDecrypter
-    ) {
+        AssertionDecrypter assertionDecrypter,
+        UserIdHashFactory userIdHashFactory,
+        MatchingServiceProxy matchingServiceClient,
+        MatchingServiceResponseDtoToOutboundResponseFromMatchingServiceMapper responseMapper) {
         return countryMetadataResolver.map(countryMetadataResolver1 ->
```

Comment:
I know that this was there before, but we can probably name the variable a bit better. This is an open source project after all and the whole world can see what our code.